transform/models/sources.yml

```diff
@@ -0,0 +1,8 @@
+version: 2 
+ 
+sources: 
+  - name: src_gharchive 
```

Comment:
Minor naming: the `src_` prefix is usual for tables but less so for source names in dbt. I'd suggest `gharchive` instead?